ResearchKit/Common/ORKImageCaptureView.m

```diff
@@ -203,6 +203,11 @@ - (UIBarButtonItem *)continueButtonItem {
 }
 
 - (void)setContinueButtonItem:(UIBarButtonItem *)continueButtonItem {
+    // If we are in an error state, then do not configure the continue button
+    if (self.error) {
+        return;
```

Comment:
And in this case, shouldn't the continueButtonItem that's been set still be saved, just not applied to the continueSkipContainer? So that if the error is cleared, you can restore the correct item.